node/src/components/consensus/era_supervisor.rs

```diff
@@ -640,11 +640,21 @@ where
                 timestamp,
                 height,
                 rewards,
+                equivocators,
                 proposer,
             }) => {
+                // If this is the era's last block, it contains rewards. Everyone who is accused in
+                // the block or seen as equivocating via the consensus protocol gets slashed.
```

Comment:
Shouldn't `accusations` and `equivocators` be equal?

Reply:
No, that's the point of this PR: The block proposer might have omitted accusations they were expected to make, because the consensus protocol knows they see someone as faulty.
By far the easiest way to fix that was to just consider those "implicitly accused" and add them in here.

Reply:
If block proposers tags someone as `Faulty` in its panorama, then that's an explicit accusation. The only problem would be if it cheated with the panorama.